ols/app/models/config.py

```diff
@@ -822,37 +822,31 @@ def __eq__(self, other: object) -> bool:
             )
         return False
 
-    def _validate_provider_and_model(
-        self, provider_attr_name: str, model_attr_name: str
-    ) -> None:
-        provider_attr_value = getattr(self.ols_config, provider_attr_name, None)
-        model_attr_value = getattr(self.ols_config, model_attr_name, None)
-
-        provider_specified = isinstance(provider_attr_value, str)
-        model_specified = isinstance(model_attr_value, str)
-
-        if provider_specified and model_specified:
-            provider_config = self.llm_providers.providers.get(provider_attr_value)
-            if provider_config is None:
-                raise InvalidConfigurationError(
-                    f"{provider_attr_name} specifies an unknown provider {provider_attr_value}"
-                )
-            model_config = provider_config.models.get(model_attr_value)
-            if model_config is None:
-                raise InvalidConfigurationError(
-                    f"{model_attr_name} specifies an unknown model {model_attr_value}"
-                )
-        elif provider_specified and not model_specified:
+    def _validate_default_provider_and_model(self) -> None:
+        selected_default_provider = self.ols_config.default_provider
+        selected_default_model = self.ols_config.default_model
+
+        provider_specified = selected_default_provider is not None
+        model_specified = selected_default_model is not None
+
+        if not provider_specified:
             raise InvalidConfigurationError(
-                f"{provider_attr_name} is specified, but {model_attr_name} is missing"
+                "default_provider with provider name is missing"
             )
-        elif not provider_specified and model_specified:
+        if not model_specified:
+            raise InvalidConfigurationError("default_model with model name is missing")
```

Comment:
should we just say "default_provider is missing" and "default_model is missing"?

Reply:
updated